Makefile

```diff
@@ -247,6 +260,20 @@ $(FLG_LINT_TTLS): $(DST_LINT)/%.linted: $(SRC_ONT)/%.ttl
       ${DOCKER_IMAGE_RUBY_RDF} validate --validate $<
 	@touch $@
 
+.PHONY: lint-jsonld
+lint-jsonld: check cache $(FLG_LINT_JSONLDS) ## Lint all JSON-LD files
+
+$(FLG_LINT_JSONLDS): $(DST_LINT)/%.linted: $(SRC_ONT)/%.jsonld
+	@echo "${COLOR_CYAN}🔬 linting: ${COLOR_GREEN}$<${COLOR_RESET}"
+	@mkdir -p -m $(PERMISSION_MODE) $(@D)
+	@cp $< $@.jsonld
+	@sed -i ${SED_FLAG} "s/\$$major/next/g" $@.jsonld
+	@docker run --rm \
+	  -v `pwd`:/usr/src/ontology:ro \
+	  -w /usr/src/ontology \
+	  ${DOCKER_IMAGE_RUBY_RDF} validate --validate $@.jsonld
+	@mv -f $@.jsonld $@
+
```

Comment:
Indeed, it is deliberate that in the context of verifying Verifiable Credentials in jsonld format, the target is the `vnext` version, which is the current latest version available. This is because the verification process retrieves resources (context) using their URIs. If, however, we were to switch to the `Major` version, which is the next one, the resources would not yet be accessible as they have not been released or deployed.